src/software/jetson_nano/battery_test.cpp

```diff
@@ -0,0 +1,15 @@
+#include <gtest/gtest.h>
+
+#include "thunderloop.h"
+
+TEST(TestBattery, is_power_stable)
+{
+    std::ifstream file("/var/log/dmesg");
```

Comment:
can you create two mock `ifstreams`? One with the error message and one without the error messages and unit tests with that